databasebuild.yaml

```diff
@@ -0,0 +1,19 @@
+# using this will require buildcomponents.yaml 
+
+build:
+  image: kevinrambaud/node-flyway:8-5.1.4-alpine
+  extends: .migrate
+  stage: build
+  services:
+    - postgres:10.6-alpine
```

Comment:
I wonder if we want to make this a variable, and if so, do variables have default ability?

Reply:
Should the value live here or in the base ci file?

Reply:
nvm